recipes-wpe/wpewebkit/wpewebkit_0.1.bb

```diff
@@ -39,7 +39,7 @@ PROVISIONING_x86 = ""
 PROVISIONING_hikey-32 = ""
 PROVISIONING_dragonboard-410c-32 = ""
 
-PACKAGECONFIG ?= "2dcanvas deviceorientation fullscreenapi encryptedmediav1 fetchapi gamepad geolocation indexeddb libinput logs mediasource notifications nativevideo sampling-profiler shadowdom subtitle subtlecrypto udev video webaudio ${WPE_PLATFORM}"
+PACKAGECONFIG ?= "2dcanvas deviceorientation fullscreenapi encryptedmediav1 fetchapi gamepad geolocation indexeddb libinput logs mediasource notifications nativevideo sampling-profiler shadowdom subtitle subtlecrypto touch udev video webaudio ${WPE_PLATFORM}"
```

Comment:
Let's not add touch to the default package config line. As that would enable it for everyone and there are currently more devices without touch then with touch capabilities. You can add your local overrides and _append it in case you need it.